protos/monocle/task_data.proto

```diff
@@ -44,3 +45,30 @@ message TaskDataGetLastUpdatedResponse {
     google.protobuf.Timestamp timestamp = 2;
   }
  }
+
+message NewTaskData {
+  google.protobuf.Timestamp updated_at = 1;
+  string change_url = 2;
+  repeated string ttype = 3;
+  string tid = 4;
+  string url = 5;
+  string title = 6;
+  string severity = 7;
+  string priority = 8;
+  int32 score = 9;
+}
+
+message AddRequest {
+  string index = 1;
+  string crawler = 2;
+  string apikey = 3;
+  repeated NewTaskData items = 4;
+}
+
+message AddResponse {
+  oneof result {
+    // TODO: define a header message with generic error
+    // then do not re-use the commit error enum.
+    TaskDataCommitError error = 1;
```

Comment:
TODO: use a better error message, e.g. using a common header field, and more specific add errors such as ToManyTaskData.